src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php

```diff
@@ -118,12 +138,137 @@ public function testItTakesMutesDeprecationFromPhpUnitFiles()
         $this->assertTrue($deprecation->isMuted());
     }
 
+    public function providerGetTypeDetectsSelf(): array
+    {
+        return [
+            'not_from_vendors_file' => [Deprecation::TYPE_SELF, '', 'MyClass1', ''],
```

Comment:
Don't pass anymore

Reply:
Cannot say I understand but I applied this change :sweat_smile:

Reply:
It's just change composer loader psr4 paths which are used to determine "self" type :)